### DISCUSSION TITLE
Remove from default list

### DISCUSSION THREAD

**Discussion by gsakkis** [community/community#53629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My main use case for lists would be to unclutter the (effectively default) list of all starred repos. So I would like to have the option to "move" (as opposed to "copy") a repository to a list so that it doesn't show up in the default list.

**Comment by pamelaacme** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> My main use case for lists would be to unclutter the (effectively default) list of all starred repos. So I would like to have the option to "move" (as opposed to "copy") a repository to a list so that it doesn't show up in the default list.

**Comment by ahmed34332** [comment]
#100370944

**Comment by ahmed34332** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> My main use case for lists would be to unclutter the (effectively default) list of all starred repos. So I would like to have the option to "move" (as opposed to "copy") a repository to a list so that it doesn't show up in the default list.

**Comment by Phuchong0259** [comment]
import OpenAI from "openai";

const openai = new OpenAI({
    apiKey: process.env.OPENAI_API_KEY,
});

const chatCompletion = await openai.chat.completions.create({
    messages: [{ role: "user", content: "Say this is a test" }],
    model: "gpt-3.5-turbo",
});

**Comment by Phuchong0259** [comment]
> msadeghm18@gmail.com

**Comment by Phuchong0259** [comment]
> https://prajwal-jaiswal.vercel.app/

**Reply by Phuchong0259** [reply]
**[]()![-5093862558183566614_121.jpg](https://github.com/community/community/assets/154827001/eefd339d-7274-4148-9909-cf2a7c37800f)

**

**Comment by Phuchong0259** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> My main use case for lists would be to unclutter the (effectively default) list of all starred repos. So I would like to have the option to "move" (as opposed to "copy") a repository to a list so that it doesn't show up in the default list.

**Comment by 1079520316** [comment]
media&info@kingdom.com.sa

**Comment by cklim24** [comment]
> msadeghm18@gmail.com

**Comment by elatmas** [comment]
https://pypodcats.live/about/

**Comment by ad0v0** [comment]
Hello!

Agree with @gsakkis. Would be really great to see this functionality. 🙏🏻  As currently it's a bit difficult to understand which repository was already added to the list, unless you remember it or click on the dropdown button with lists.
